docs/development.md

```diff
@@ -54,9 +54,14 @@ vagrant up centos7-devel
 The box can now be accessed via ssh and the Rails server started directly (this assumes you are connecting as the default `vagrant` user):
 
 ```sh
+cd forklift
+vagrant ssh centos7-devel -- -L3000:localhost:3000
 vagrant ssh centos7-devel
 cd foreman
-bundle exec foreman start
+rake permissions:reset
+npm install
+rake webpack:compile
+bundle exec foreman start -p 3000
```

Comment:
Default port is 3000 already, no?

Reply:
5000 in rails 5 unless you have a .env specifying port (which puppet-katlelo_devel does now)